station/receiver.py

```diff
@@ -89,9 +89,10 @@ def cmd():
             rating = get_rating_for_product(product[1], satellite.get("rate"))
             logging.info("Product %s got rating %s" % (product[1], rating))
             # TODO: Submit ALL products and logs
+            files = [ product[1] ]
```

Comment:
Maybe, we should send all products?

Reply:
That's a good and a bit risky idea. Do you want to upload 150MB+ wav files? So maybe not all, but most. In general, I totally agree it's a great idea. Created #66 for this. It's way too large for this ticket.